### PR TITLE
HASSIO Improved addon view

### PR DESCRIPTION
**Pull request by ciotlosm**
- [x] Show addon status based on icon (running/stopped)
- [x] Use more fluid layout

Note: 
Addon version is missing because it cluttered tiles and mostly used in troubleshooting (detailed view) than in an overview

screen shot 2018-02-06 at 22 53 20
screen shot 2018-02-07 at 00 27 53

@c727

### CODE REVIEW

**Comment by c727** [comment]
hey everyone, please shut on eye if you review the hassio-style file as we havn't removed everything we need for future PRs

(adding this to @balloob to-do list)

**Comment by balloob** [comment]
The icons look off-centered when there are only 2 lines of text. Any possibility of shifting that a little up so it is centered between the first 2 lines?

File: src/components/hassio-card-content.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        float: left;
+      }
+      iron-icon.update {
+        color: var(--paper-orange-400);
+      }
+      iron-icon.running,
+      iron-icon.installed {
+        color: var(--paper-green-400);
+      }
+      iron-icon.primary {
```
Comment (balloob) [inline src/components/hassio-card-content.html]: The name primary here seems weird. All other class names are describing what it is for (update/running), this one describes what it does (primary color)

File: src/components/hassio-card-content.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    </style>
+    <iron-icon icon='[[icon]]' class$='[[iconClass]]' title='[[iconTitle]]'></iron-icon>
+    <div>
+      <div class='title'>[[title]]</div>
+      <div class='description'>
+        <template is='dom-if' if='[[!datetime]]'>
+          [[description]]
+        </template>
+        <template is='dom-if' if='[[datetime]]'>
+          <ha-relative-time datetime='[[description]]'></ha-relative-time>
```
Comment (balloob) [inline src/components/hassio-card-content.html]: This is weird. Don't change the meaning of the value of a variable based on a boolean. Why not just have datetime be a string and pass that to `ha-relative-time` when available?

**Review by balloob — APPROVED** [review]

File: src/resources/hassio-style.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<dom-module id='hassio-style'>
+  <template>
+    <style>
+      paper-card {
```
Comment (balloob) [inline src/resources/hassio-style.html]: Using elements as CSS selector in a reusable stylesheet is going to bite you in the future.

**Comment by c727** [comment]
OK after we fixed the icon resize issue

File: hassio/dashboard/hassio-addons.html [hunk 4]
```diff
@@ -1,39 +1,40 @@
...
-<link rel="import" href="../../bower_components/paper-item/paper-item-body.html">
+<link rel='import' href='../../bower_components/polymer/polymer-element.html'>
+<link rel='import' href='../../bower_components/paper-card/paper-card.html'>
 
 <link rel='import' href='../../src/util/hass-mixins.html'>
+<link rel='import' href='../../src/components/hassio-card-content.html'>
+<link rel='import' href='../../src/resources/hassio-style.html'>
 
-<dom-module id="hassio-addons">
+<dom-module id='hassio-addons'>
```
Comment (balloob) [inline hassio/dashboard/hassio-addons.html]: Just a heads up, your quote preference is making code review more difficult as it's no longer clear what things are changes and which ones are style tweaks.

Comment (c727) [reply hassio/dashboard/hassio-addons.html]: Ok, I will avoid this for other PRs

**Comment by c727** [comment]
we will fix the alignment issues with the icon and the text late, help wanted

**Comment by balloob** [comment]
@c727 make sure to coordinate with Pascal about updating the Hass.io panel in a release.

**Pull request merged**